units/ArmGantry/armraz.lua

```diff
@@ -118,7 +118,10 @@ return {
 				cratermult = 0,
 				edgeeffectiveness = 0.15,
 				explosiongenerator = "custom:laserhit-medium-red",
+				energypershot=75,
```

Comment:
Was this line supposed to be here?

Reply:
Yes, it's a old mistake that it isn't already there (and double yes Zecrus approves the change)